src/app/components/ResultsList/ResultsList.jsx

```diff
@@ -109,86 +109,87 @@ const ResultsList = ({
-        <ElectronicResourcesResultsItem
-          resources={eResources}
-          onClick={resourcesOnClick}
-          bibUrl={bibUrl}
-        />
-        {
-          hasRequestTable &&
-          <>
-            <ItemTable
-              items={items.slice(0, itemTableLimit)}
-              bibId={bibId}
-              id={null}
-              searchKeywords={searchKeywords}
-              page="SearchResults"
+              to={bibUrl}
+              className="title">
+              {bibTitle}
+            </Link>
+          </CardHeading>
+          <CardContent className="nypl-results-item-description">
+            <ul>
+              <li className="nypl-results-media">{materialType}</li>
+              <li className="nypl-results-publication">{publicationStatement}</li>
+              {yearPublished}
+              {
+                totalItems > 0 ?
+                  <li className="nypl-results-info">
+                    {itemMessage}
+                  </li>
+                  : ''
+              }
+            </ul>
+            <ElectronicResourcesResultsItem
+              resources={eResources}
+              onClick={resourcesOnClick}
+              bibUrl={bibUrl}
             />
             {
-              totalPhysicalItems > 3 ?
-              (
-                <Link
-                  onClick={resourcesOnClick}
-                  to={`${bibUrl}#items-table`}
-                  className="search-results-list-link"
-                  id="physical-items-link"
-                >
-                  <Text isBold size="caption">
-                    {`See all ${totalPhysicalItems} in Library & Offsite Items`} <RightWedgeIcon />
-                  </Text>
-                </Link>
-              ) :
-              null
+              hasRequestTable &&
+              <>
+                <ItemTable
+                  items={items.slice(0, itemTableLimit)}
+                  bibId={bibId}
+                  id={null}
+                  searchKeywords={searchKeywords}
+                  page="SearchResults"
+                />
+                {
+                  totalPhysicalItems > 3 ?
+                    (<CardAction>
+                      <Link
+                        onClick={resourcesOnClick}
+                        to={`${bibUrl}#items-table`}
+                        className="search-results-list-link"
+                        id="physical-items-link"
+                      >
+                        <Text isBold size="caption">
+                          {`View all ${totalPhysicalItems} in Items`} <RightWedgeIcon />
+                        </Text>
+                      </Link>
+                    </CardAction>
+                    ) :
+                    null
+                }
+              </>
             }
-          </>
-        }
+          </CardContent>
+        </Card>
       </li>
     );
   };
 
   const resultsElm = results.map((bib, i) => generateBibLi(bib, i));
 
   return (
-    <ul
+    <li
```

Comment:
This should be the `ul` since it's the parent. The `resultsElm` variable contains the array of `li`s. This is where the `List` component can replace the `ul` but that's not necessary.